web/src/hooks/redux.js

```diff
@@ -55,7 +55,10 @@ export function useCurrentZoneData() {
       return null;
     }
     if (zoneTimeIndex === null) {
-      return grid.zones[zoneId];
+      // Return latest history data unless latest history does not correspond to the current grid datetime
+      return (zoneHistory[zoneHistory.length - 1].stateDatetime === grid.datetime
+        ? zoneHistory[zoneHistory.length - 1]
+        : null);
```

Comment:
This change also broke how the mockserver is implemented.. We should either in the mockserver manually override stateDatetime in the response, or consider if we can avoid having this check? 🤔

Reply:
Ah good point, I only added this check while finally working with production data.

I believe the mockserver also has the issue with the state being somewhere in 2017, whereas the history file is recent. So we could perhaps also just update the mockserver files, as it is a fair assertion that the state and history files are somewhat related.

On the other hand, it is not like returning null really fixes anything if the files do become out of sync, so I am down with removing the check as well, lmk which one you'd prefer